fetch.bs

```diff
@@ -3741,13 +3740,6 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
 
   <p>When in doubt, return a <a>network error</a>.
 
- <dt>"<code>filesystem</code>" <!-- flag below also applies to "indexeddb" -->
- <dd>
-  <p>If <var>request</var>'s <a>sandboxed-storage-area-URLs flag</a> is set,
```

Comment:
Could you also remove the definition of "sandboxed-storage-area-URLs flag"?

Reply:
you got it